### ISSUE TITLE
Is this still working?

### ISSUE THREAD

**Issue by kennanseno** [jballant/webpack-strip-block#9]
Hi. I've been trying to get this to work on my angular project with webpack but just cant seem to get it working. is this still being maintained or working for angular projects?

**Comment by jballant** [comment]
Hello, I just tested the loader with webpack 2.4.1, 2.7.0 and 3.8.1 on node 6.10.2. Can you tell me which version of webpack are you using? Also, if you can provide a code snippet and the configuration you are using for the loader, that would be helpful.

**Comment by kennanseno** [comment]
I'm using 3.8.1 too. not sure if this works on angular projects though?

**Comment by jballant** [comment]
I haven't tried it with an Angular project. What is different about how an Angular project uses webpack? Is this Angular 2+? Is the project written in Typescript?

**Comment by kennanseno** [comment]
@jballant i'm using the latest angular 5 and yes, it is written in typescript. I'm still pretty new to webpack and angular so still not sure if it's my config that is the problem.

**Comment by kennanseno** [comment]
I'm basically trying to remove and import line of code before anything else so angular will not bundle it up during AOT and from what I understood loaders are run from bottom-to-top so I tried putting it before the tree-shaking and bundling but it is still not removed for some reason.

Here's my config file(https://www.pastiebin.com/5a0d75475daa0) if you could help know where I should put it, I'd greatly appreciate it!

**Comment by jballant** [comment]
I've tested the `webpack-strip-block` to remove ES6 imports, and it seems to work.

My main concern would be the AngularCompilerPlugin. I'm not exactly sure how it works, but if it re-writes the imports at all before Webpack loaders are invoked, then the `webpack-strip-block` loader might not work.

Perhaps you can try testing the `webpack-strip-block` loader in your config without the two `@angular-devkit/build-optimizer/webpack-loader` declarations. There is also the `'@ngtools/webpack'` loader that pairs with AngularCompilerPlugin. 

Additionally, you can try `enforce: pre` in your rule declaration (https://webpack.js.org/configuration/module/#rule-enforce). But again, if the AngularCompilerPlugin is rewriting import statements at all before any loaders can be invoked, then the `webpack-strip-block` loader might not work.

**Comment by kennanseno** [comment]
yes, that's what most likely is happening that angularCompilerPlugin does it's thing first before the webpack-strip-block which is confusing since I thought loaders run first before the plugins?

**Comment by stevethemacguy** [comment]
@kennanseno: Did you ever find a solution? I'm trying to do the same thing with no luck.

**Comment by stevethemacguy** [comment]
@kennanseno. I was having a similar problem and solved it using process.env.NODE_ENV. See https://stackoverflow.com/questions/52654563/how-to-prevent-typescript-code-blocks-from-being-compiled-by-the-angular-aot-web
